### DISCUSSION TITLE
Corsair Elite Capellix AIO

### DISCUSSION THREAD

**Discussion by RexDomi** [Rem0o/FanControl.Releases#696] [Q&A]
Anyone tried this software on one of the corsair elite capellix aios? the pump is kinda loud without the option to turn it down in ique. was wondering if i could turn it down using fan control.

**Comment by MrShaky** [comment]
I also came here in search of something to replace the travesty that is iCUE for my H115i.  I doubt there are many significant differences between our hardware, so it's safe to say that the pump along with the LEDs and fans are controlled from the built-in USB device.

Currently, this tool does not have any way to interface with that USB device, but the developer that created the Commander plugin suggested adding AIO controls might be possible.  If you'd like to have an interim solution as I did, there is SIV; it was referenced in one of many community threads related to iCUE4 woes...  it can be found at http://rh-software.com/

**Reply by ghost** [reply]
It is possible to at least read the AIO pump speed, the latest [beta](https://github.com/iJacks1980/FanControl.CommanderPRO/releases/tag/v1.0.2a) of my plugin however leaves the pump out.  Right now it's _all_ read only, but I am not sure that controlling the AIO pump is a good idea...will test it on my hardware first though.

**Reply by MrShaky** [reply]
Based on my very brief experience with SIV, it looks like the program does indeed use those read-only locations you've mentioned for monitoring, but allows AIO pump and fan control through registry settings...
